packages/block-library/src/style.native.scss

```diff
@@ -0,0 +1,62 @@
+// This tag marks the start of the styles that apply to editing canvas contents and need to be manipulated when we resize the editor.
+#start-resizable-editor-section {
+	display: none;
+}
+
+// Background colors.
+@include background-colors();
+
+// Foreground colors.
+@include foreground-colors();
+
```

Comment:
`@include gradient-colors()` is currently failing with `error: bundling failed: ../block-library/src/style.native.scss: Failed to parse declaration "background: linear-gradient(135deg, #0693e3 0%, #9b51e0 100%)"`